app/models/requests/MovementDetailsRequest.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.requests
+import play.api.libs.json.{Json, OFormat}
+
+case class MovementDetailsRequest(dateTime: String)
```

Comment:
Why DateTime is string ?

Reply:
`Instant` would be better :)

Reply:
We do not need to send an instant, because we are not storing this information to the DB. All we are doing is converting the data we collected from the FE pages, which will return a String and then submit that to the BE and the BE will relay. that messages (converting into XML).
Yes, I would otherwise convert it to Instant if there was a need.